### PR TITLE
MGDOBR-33: pom fix

### PR DESCRIPTION
**Pull request by r00ta**
Fixes https://issues.redhat.com/browse/MGDOBR-33

### CODE REVIEW

File: build-parent/pom.xml [hunk 1]
```diff
@@ -2,17 +2,16 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <parent>
-    <artifactId>event-connect</artifactId>
+    <artifactId>event-bridge</artifactId>
     <groupId>com.redhat.developer</groupId>
     <version>0.0.1-SNAPSHOT</version>
-    <relativePath>../pom.xml</relativePath>
```
Comment (r00ta) [inline build-parent/pom.xml:8]: This is the fix for the wrong order build

File: 0 [hunk 2]
```diff
@@ -0,0 +1 @@
+11
```
Comment (danielezonca) [inline 0:1]: To be removed

File: pom.xml [hunk 3]
```diff
@@ -4,18 +4,38 @@
...
+  <artifactId>event-bridge</artifactId>
   <version>0.0.1-SNAPSHOT</version>
   <packaging>pom</packaging>
-  <name>Event Connect</name>
-  <description>Parent module for event connect</description>
+
+  <name>Event Bridge</name>
+  <description>Parent module for event bridge</description>
+
+  <inceptionYear>2019</inceptionYear>
```
Comment (danielezonca) [inline pom.xml:14]:
```suggestion
  <inceptionYear>2021</inceptionYear>
```

File: pom.xml [hunk 4]
```diff
@@ -4,18 +4,38 @@
...
-  <description>Parent module for event connect</description>
+
+  <name>Event Bridge</name>
+  <description>Parent module for event bridge</description>
+
+  <inceptionYear>2021</inceptionYear>
+  <organization>
+    <name>JBoss by Red Hat</name>
+    <url>http://www.jboss.org/</url>
+  </organization>
```
Comment (danielezonca) [inline pom.xml:18]:
```suggestion
  <organization>
    <name>Red Hat</name>
    <url>http://www.redhat.com/</url>
  </organization>
```

**Review by danielezonca — APPROVED** [review]

**Pull request merged**